syz-manager/rpc.go

```diff
@@ -116,109 +105,31 @@ func (serv *RPCServer) Connect(a *rpctype.ConnectArgs, r *rpctype.ConnectRes) er
 	serv.mu.Lock()
```

Comment:
`serv.coverFilter = coverFilter` should also be under the mutex, since it's a shared var.